resources/js/controllers/fields/quill_controller.js

```diff
@@ -76,6 +76,25 @@ export default class extends Controller {
         ];
     }
 
+    containerCustomToolbar() {
```

Comment:
What do you think about not duplicating definitions but using a single method named `containerToolbar` ?

Reply:
hmmm, and use [this](https://github.com/orchidsoftware/platform/pull/1254#discussion_r496589195) as default?

Reply:
Yes, when we define the default values, we won't need to duplicate them.